build_tools/bazel_to_cmake/bazel_to_cmake.py

```diff
@@ -151,13 +151,20 @@ def convert_directory(directory_path, write_files, allow_partial_conversion,
   if verbosity >= 1:
     log(f"Processing {rel_dir_path}")
 
-  build_file_path = os.path.join(directory_path, "BUILD")
-  cmakelists_file_path = os.path.join(directory_path, "CMakeLists.txt")
-
-  rel_cmakelists_file_path = repo_relpath(cmakelists_file_path)
-  rel_build_file_path = repo_relpath(build_file_path)
-
-  if not os.path.isfile(build_file_path):
+  # Scan for a BUILD file.
+  build_file_found = False
+  build_file_basenames = ["BUILD", "BUILD.bazel"]
+  for build_file_basename in build_file_basenames:
+    build_file_path = os.path.join(directory_path, build_file_basename)
+    cmakelists_file_path = os.path.join(directory_path, "CMakeLists.txt")
+
+    rel_cmakelists_file_path = repo_relpath(cmakelists_file_path)
+    rel_build_file_path = repo_relpath(build_file_path)
+    if os.path.isfile(build_file_path):
+      build_file_found = True
+      break
```

Comment:
Shouldn't need the cmake file variable in the loop

```suggestion
  # Scan for a BUILD file.
  build_file_found = False
  build_file_basenames = ["BUILD", "BUILD.bazel"]
  for build_file_basename in build_file_basenames:
    build_file_path = os.path.join(directory_path, build_file_basename)

    rel_build_file_path = repo_relpath(build_file_path)
    if os.path.isfile(build_file_path):
      build_file_found = True
      break
  cmakelists_file_path = os.path.join(directory_path, "CMakeLists.txt")
  rel_cmakelists_file_path = repo_relpath(cmakelists_file_path)
```